scene/animation/animation_tree.h

```diff
@@ -197,9 +197,13 @@ class AnimationTree : public Node {
 		bool loc_used = false;
 		bool rot_used = false;
 		bool scale_used = false;
+		Vector3 init_loc = Vector3();
+		Quaternion init_rot = Quaternion();
+		Quaternion init_rot_accum = Quaternion();
```

Comment:
I wondered if `Quaternion init_rot_accum` was necessary as a member, but since `init_rot.log()` calculation is required for this every time at line 1164 of animation_tree.cpp, I defined it to reduce the calculation cost.